### PR TITLE
feat(Timeline): rename name to title & date to subtitle with backwards compatibility

### PR DESCRIPTION
**Pull request by thaytharma**
Rename name/dare-props in`Timeline`-component:
- `name` is changed to `title` in TimelineItem
- `date` is changed to `subtitle` in TimelineItem

### CODE REVIEW

File: packages/dnb-eufemia/src/components/timeline/style/_timeline.scss [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
       }
-      &__name {
+      &__title {
         cursor: text;
       }
     }
     &__content {
       padding-bottom: var(--spacing-small);
-      &__date {
+      &__subtitle {
```
Comment (langz) [inline packages/dnb-eufemia/src/components/timeline/style/_timeline.scss:31]: I think we need to update the snapshot tests for a change like this.
`yarn test:update`

**Review by langz — APPROVED** [review]
Looks great 👍 

Not sure which release this should be included in if it's "breaking", based on semantic versioning 🤔

File: packages/dnb-eufemia/src/components/timeline/style/_timeline.scss (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/timeline/style/_timeline.scss:31]: Me and @thaytharma have agreed to serve backwards compatibility and release it as a fix or feature. 

This PR needs to be squashed to a single commit with another git prefix. 

Sidenote: for a breaking change, we need more in the commit message than just the break prefix. More details [here](https://eufemia.dnb.no/contribute/style-guides/git).

**Review by tujoworker — APPROVED** [review]
Thanks again for making this PR happen 🙏

I have added backwards compatibility (+ some more) in this commit: https://github.com/dnbexperience/eufemia/pull/1468/commits/a7cd72b0bbc56391bbadea63a2926e0c8a192e7e

**Pull request merged**